sdk/agrifood/azure-agrifood-farming/setup.py

```diff
@@ -70,6 +70,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.14.0",
```

Comment:
`azure.core.rest` was GA'ed in version 1.20.0, so you'll need to bump the minimum version (and make a note in your changelog) to avoid the missing module error in pipelines: https://dev.azure.com/azure-sdk/public/_build/results?buildId=2176488&view=logs&j=4fa73ede-de59-50b0-851d-db96582ea108&t=271dca1a-6c62-5d1b-458d-2dd10be9b8db&l=5821
```suggestion
        "azure-core<2.0.0,>=1.20.0",
```